code/index.html

```diff
@@ -2,21 +2,93 @@
 <html>
   <head>
     <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Ylva Nilsson" />
+    <meta name="description" content="A News Site about sailing boats" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
     <header>
       <!-- Some kind of logo and navbar -->
+      <div class="site-logo-name">
+        <img class="logo" src="./icons/icons8-sailing-ship-96.png" alt="logo" />
+        <span>SKEPP OHOJ!</span>
+      </div>
+
+      <nav>
+        <a href="#">HOME</a>
+        <a href="#">ABOUT</a>
+        <img class="logo" src="./icons/icons8-sailing-ship-96.png" alt="logo" />
```

Comment:
You could have the image once and re-position it with `position: absolute;` instead of having the same image twice :)